types/serde.go

```diff
@@ -17,11 +17,24 @@ func (d *Destination) UnmarshalText(text []byte) error {
 	return nil
 }
 
+type JsonRpcRequest struct {
```

Comment:
Slightly out of scope for this PR: I'm not super clear why we want to split the types across this file and `rps/serde/jsonrpc.go`. 

Do you think we could tidy this up (even in a follow-up PR?).

Reply:
Let's discuss at standup. It might be nice encapsulation for the node to not know anything about json rpc?